Diffusion.xcodeproj/project.pbxproj

```diff
@@ -704,6 +722,8 @@
 				ENABLE_PREVIEWS = YES;
 				GENERATE_INFOPLIST_FILE = YES;
 				INFOPLIST_FILE = Diffusion/Info.plist;
+				INFOPLIST_KEY_CFBundleDisplayName = Diffusers;
```

Comment:
Same comment for `CFBundleDisplayName`

Reply:
Same answer. I'm not sure why they got in here.